### PR TITLE
Fix broken File Adapters link

### PR DESCRIPTION
**Pull request by androidi**
File Adapters link on File (Api) -page of documentation is broken.

### CODE REVIEW

File: packages/fields/src/types/File/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ keystone.createList('Applicant', {
 
 | Option       | Type      | Default  | Description                                                                                            |
 | ------------ | --------- | -------- | ------------------------------------------------------------------------------------------------------ |
-| `adapter`    | `Object`  | Required | See the [File Adapters](https://www.keystonejs.com/keystone/file-adapters/) page for more information. |
+| `adapter`    | `Object`  | Required | See the [File Adapters](/keystonejs/file-adapters/) page for more information. |
```
Comment (MadeByMike) [inline packages/fields/src/types/File/README.md]:
```suggestion
| `adapter`    | `Object`  | Required | See the [File Adapters](https://www.keystonejs.com/keystonejs/file-adapters/) page for more information. |
```

Comment (MadeByMike) [reply packages/fields/src/types/File/README.md]: We use absolute URLs in README.MD files so they work on github.

Comment (androidi) [reply packages/fields/src/types/File/README.md]: Ok, I found both absolute and relative URLs, so I opted for relative... check e.g. file packages/fields/README.md, there are both absolute and relative URLs in there.

Comment (MadeByMike) [reply packages/fields/src/types/File/README.md]: Good catch, those should be absolute as well.

**Pull request closed without merging**